File: Python/Data Structures & Algorithms Module - Python/Week1/Person.py
"""
Person.py
Billy Cussen
09/11/2020
"""

class myPerson:

    def __init__(self, firstName, secondName, age):
        self.firstName = firstName
        self.secondName = secondName
        self.age = age

    def __str__(self):
        return "First Name: "+str(self.firstName)+", Second Name: "+str(self.secondName)+", Age: "+str(self.age)

File: Java/Data Structures & Algorithms Module - Java/Week8/Revision/Week1/Person.java
/*
Person.java
Billy Cussen
09/11/2020
*/
public class Person{

    private String firstname;
    private String secondname;
    private int age;

    public Person(String firstname, String secondname, int age){
        this.firstname = firstname;
        this.secondname = secondname;
        this.age = age;
    }

    @Override
    public String toString(){
        return "First Name: "+this.firstname+", Second name: "+this.secondname+", Age: "+this.age;
    }

}